JavaScriptSPA/ui.js

```diff
@@ -11,14 +12,13 @@ function showWelcomeMessage(account) {
     // Reconfiguring DOM elements
     cardDiv.style.display = 'initial';
     welcomeDiv.innerHTML = `Welcome ${account.name}`;
-    signInButton.nextElementSibling.style.display = 'none';
-    signInButton.setAttribute("onclick", "signOut();");
-    signInButton.setAttribute('class', "btn btn-success")
-    signInButton.innerHTML = "Sign Out";
+    signInButton.style.display = 'none';
+    signOutButton.style.display = 'initial';
```

Comment:
Instead of manipulating `style`, add/remove classes (in this case, `d-none`) from `signInButton.classList`